core/src/main/scala/cats/data/Ior.scala

```diff
@@ -28,6 +28,11 @@ sealed abstract class Ior[+A, +B] extends Product with Serializable {
     case Ior.Both(a, b) => fab(a, b)
   }
 
+  final def putLeft[AA >: A](left: AA): AA Ior B =
```

Comment:
actually, why does this need to be `AA >: A` and not just `AA`? Aren't we fully replacing? Same with `putRight`?